cmd/controller/app/controller.go

```diff
@@ -0,0 +1,87 @@
+/*
```

Comment:
It's not new, per se, but it's changed enough that git probably doesn't recognize it as the same file. I don't pretend to know git internals to that level.